yarn.lock

```diff
@@ -1698,11 +1698,6 @@ chokidar@^3.0.0, chokidar@^3.0.2, chokidar@^3.3.1:
   optionalDependencies:
     fsevents "~2.1.2"
 
-chownr@^1.1.1:
```

Comment:
Undo all your changes to yarn.lock. Just running `yarn install` should not change this file, although maybe it's a Windows compatibility thing.